pinot-core/src/main/java/com/linkedin/pinot/core/segment/creator/impl/SegmentColumnarIndexCreator.java

```diff
@@ -197,9 +197,13 @@ public void init(SegmentGeneratorConfig segmentCreationSpec, SegmentIndexCreatio
         continue;
       }
 
-      OffHeapBitmapInvertedIndexCreator invertedIndexCreator =
-          new OffHeapBitmapInvertedIndexCreator(file, cardinality, totalDocs, totalNumberOfEntries, fieldSpec);
-      invertedIndexCreatorMap.put(column, invertedIndexCreator);
+      try {
```

Comment:
Why do we need this try catch and swallow the exception?

Reply:
Because we want to attempt to build inverted index for other columns if possible. See line 169, it is a loop.